crates/hotshot-stake-table/src/vec_based.rs

```diff
@@ -255,14 +255,15 @@ where
     #[must_use]
     pub fn new(capacity: usize) -> Self {
         let bls_comm_preimage = vec![F::default(); capacity * <K1 as ToFields<F>>::SIZE];
-        let default_bls_comm =
-            VariableLengthRescueCRHF::<F, 1>::evaluate(&bls_comm_preimage).unwrap()[0];
+        let default_bls_comm = VariableLengthRescueCRHF::<F, 1>::evaluate(&bls_comm_preimage)
+            .expect("Couldn't evaluate bls commitment preimage")[0];
```

Comment:
Is `new` only called by test code? If not, it's probably better to return an error rather than use `expect`.